kb_sdk_plus.md

```diff
@@ -1,10 +1,98 @@
 # KB-SDK plus
 
+## Why?
+
+The KBase SDK is a core piece of infrastructure, but it has become exceedingly difficult to
+maintain and expand as the test suite is exceptionally complex and currently does not pass.
+As such, large changes to the codebase are risky and no updates have been released for 5+ years.
+Furthermore, due to outdated dependencies, the SDK will not run on modern machines.
+
+To rectify this while still supporting older apps, we have elected to fork the
+https://github.com/kbase/kb_sdk repo to `kb_sdk_plus`. Version 1.2.1 is the last `kb_sdk`
+release and will be kept as is. Future SDK development will occur in `kb_sdk_plus`.
+
+## Why not kb_sdk2?
+
+`<repo>2` generally signifies a greenfield rewrite of a codebase in KBase,
+which `kb_sdk_plus` is not. It's forking a codebase to leave the original codebase untouched
+and runnable while we make changes to the forked codebase.
+
+## Guidelines
+
+* Backwards incompatible changes are expected in some respects:
+  * R code generation will be removed
+    * Unused in KBase or anywhere else as far as we know
```

Comment:
There are at least two R KBase apps, but at least one, possibly both, are called from the command line so the fact that they are in R is irrelevant.

Reply:
the only R app at https://narrative.kbase.us/legacy/catalog/modules is Roman's ContigFilter test app and it's in dev, so I think we can safely remove R support.

Reply:
Added a note that one test app does exist for accuracy